lib/ansible/modules/network/nxos/nxos_vlan.py

```diff
@@ -254,10 +254,10 @@ def get_vlan(vlanid, module):
 
 def apply_key_map(key_map, table):
     new_dict = {}
-    for key, value in table.items():
+    for key in table:
         new_key = key_map.get(key)
         if new_key:
-            new_dict[new_key] = str(value)
+            new_dict[new_key] = str(table.get(key))
```

Comment:
I don't really have a problem with this one as it is... it's arguably the cleanest of the versions I've seen, which makes me start to question how it got to the version we've seen elsewhere.